tests/test/import/test.sh

```diff
@@ -34,6 +35,65 @@ rlJournalStart
         rlAssertNotGrep '/usr/share/rhts-library/rhtslib.sh' 'runtest.sh'
     rlPhaseEnd
 
+    rlPhaseStartTest 'Import Restraint metadata'
+        rlRun 'tmt test import --restraint --no-nitrate | tee output'
+	rlAssertGrep 'Restraint file found in' 'output'
+        rlAssertGrep 'summary: Simple smoke test using restraint' 'main.fmf'
+        rlRun 'grep -A1 recommend main.fmf | grep fmf'
+	rlAssertGrep 'test: ./runtest.sh' 'output'
+	rlAssertGrep 'duration: 6m' 'main.fmf'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Import both Makefile and Restraint metadata. Expect Restraint to be used.'
+        rlRun 'tmt test import --makefile --restraint --no-nitrate | tee output'
+        rlAssertGrep 'Restraint file found in' 'output'
+        rlAssertGrep 'summary: Simple smoke test using restraint' 'main.fmf'
+        rlRun 'grep -A1 recommend main.fmf | grep fmf'
+        rlAssertGrep 'test: ./runtest.sh' 'output'
+        rlAssertGrep 'duration: 6m' 'main.fmf'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Import specifying not to use Makefile. Expect Restraint to be used.'
+        rlRun 'tmt test import --no-makefile --no-nitrate | tee output'
+        rlAssertGrep 'Restraint file found in' 'output'
+        rlAssertGrep 'summary: Simple smoke test using restraint' 'main.fmf'
+        rlRun 'grep -A1 recommend main.fmf | grep fmf'
+        rlAssertGrep 'test: ./runtest.sh' 'output'
+        rlAssertGrep 'duration: 6m' 'main.fmf'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Import specifying not to use Makefile or Restraint. Expect Restraint to be used.'
+        rlRun 'tmt test import --no-makefile --no-restraint --no-nitrate | tee output'
+        rlAssertGrep 'Restraint file found in' 'output'
+        rlAssertGrep 'summary: Simple smoke test using restraint' 'main.fmf'
+        rlRun 'grep -A1 recommend main.fmf | grep fmf'
+        rlAssertGrep 'test: ./runtest.sh' 'output'
+        rlAssertGrep 'duration: 6m' 'main.fmf'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Import metadata'
+        rlRun 'tmt test import --no-nitrate | tee output'
+        rlAssertGrep 'Makefile found in' 'output'
+        rlAssertGrep 'summary: Simple smoke test' 'main.fmf'
+        rlRun 'grep -A1 require main.fmf | grep tmt'
+        rlRun 'grep -A1 recommend main.fmf | grep fmf'
+    rlPhaseEnd
+
+
+    rlPhaseStartTest 'Verify error returned when no Makefile exists.'
+        rlRun "mv $tmp/data/parent/child/Makefile $tmp/data/parent/child/.Makefile" 0 "Removing Makefile in tmp directory"
+	rlRun 'tmt test import --no-nitrate 2>&1 | tee output' 2
+        rlAssertGrep 'Unable to find Makefile' 'output'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Verify error returned when no Restraint metadata file exists.'
+        rlRun "mv $tmp/data/parent/child/metadata $tmp/data/parent/child/.metadata" 0 "Removing restraint metadata file in tmp directory"
```

Comment:
There is rlFileBackup & rlFileRestore in beakerlib. So you might call `rlFileBackp <path>`, then `rm <path>` and in the end just call `rlFileRestore` to restore all backuped files.